portfolio/src/main/java/com/google/sps/Puzzle.java

```diff
@@ -16,7 +16,13 @@
   /*
   * Difficulty of the puzzle
   */
-  private String difficulty;
+  enum difficultyEnum {
+      Easy,
+      Medium,
+      Hard
+  }
+
+  private difficultyEnum difficulty;
```

Comment:
The Java pattern is to use camel case for the enum name and capitalized underscores for the enum values:
```suggestion
  enum Difficulty {
      EASY,
      MEDIUM,
      HARD
  }
  
  private Difficulty difficulty;
```